app/lib/constants/api/api_const.dart

```diff
@@ -3,6 +3,9 @@ class ApiConst {
   static const signUp = '$domain/auth/sign_up';
   static const home = '$domain/report/dashboard';
   static const hatim = '$domain/hatim/join_to_hatim';
+  static Uri audioBgImage = Uri.parse(
+    'https://github.com/Eldar2021/my_quran/blob/main/app/assets/images/app_icon.png?raw=true',
+  );
```

Comment:
```suggestion
  static const appLogoLink = 'https://github.com/Eldar2021/my_quran/blob/main/app/assets/images/app_icon.png?raw=true';
```

user `Uri.parse(Apiconst.appLogoLink)`